Search-algorithms/src/base/SorterAlgorithm.java

```diff
@@ -4,7 +4,9 @@
 
 	public abstract Integer[] sorting(Integer[] arrayToBeOrdered);
 
-	public abstract Integer[] sorting(Integer[] v, Integer[] b,Integer maxValue);
+	public abstract Integer[] sorting(Integer[] arrayToBeOrdered, Integer[] OrderedArray,Integer maxValue);
```

Comment:
il secondo parametro non è necessario, il vettore ordinato viene restituito in output

Reply:
il counting sort ha bisogno di quel riferimento  per potere ordinare effettivamente il vettore.
ho bisogno di un metodo con quella firma